src/main/scala/no/ndla/auth/model/Me.scala

```diff
@@ -0,0 +1,11 @@
+/*
+ * Part of NDLA auth.
+ * Copyright (C) 2017 NDLA
+ *
+ * See LICENSE
+ */
+
+package no.ndla.auth.model
+
+
+case class Me (name: String)
```

Comment:
Kanskje. Om det er flere ting vi har behov for i frontend. Mulig den forsvinner også.